e2e.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -Eeuxo pipefail
+
+prepare_host() {
+    sudo apt -y update
+    sudo apt -y install linux-modules-extra-$(uname -r)
+    sudo mount --make-shared /sys
+    sudo mount --make-shared /
+    sudo mount --make-shared /dev
+}
+
+run_kind() {
+    echo "Download kind binary..."
```

Comment:
could prevent downloading of the binaries by checking if they're already there (and possibly expected version)?

Reply:
Lets do this separately and not hold this PR for that. This is just taken from other repos.
This check must also be added to e2e.sh script of operator and charts repos.